indico/modules/events/layout/util.py

```diff
@@ -349,6 +378,39 @@ def get_css_url(event, force_theme=None, for_preview=False):
         return _build_css_url(layout_settings.get(event, 'theme'))
 
 
+def _build_js_url(theme):
+    if ':' not in theme:
+        return None
+    try:
+        if not (path := get_plugin_conference_themes()[theme].js_path):
+            return None
+    except KeyError:
+        return None
+    plugin = theme.split(':', 1)[0]
+    return url_for_plugin(plugin + '.static', filename=path)
```

Comment:
I'd change this to an f-string here and also in `_build_css_url`:

```suggestion
    return url_for_plugin(f'{plugin}.static', filename=path)
```